docs/appkit/react/solana/implementation.mdx

```diff
@@ -4,7 +4,7 @@ import TabItem from '@theme/TabItem'
 On top of your app set up the following configuration, making sure that all functions are called outside any React component to avoid unwanted rerenders.
 
 ```tsx
-import { createAppKit, defaultConfig } from '@web3modal/solana/react'
+import { createAppKit, defaultConfig } from '@reown/solana/react'
```

Comment:
```suggestion
import { createAppKit } from '@reown/appkit/react'
```

You can only import `createAppKit` from `@reown/appkit` or `reown/appkit/react`. I am unable to find `defaultConfig` inside this package. Can you please check?